Alpaca.Markets/Messages/JsonStreamQuote.cs

```diff
@@ -33,16 +33,18 @@ internal class JsonStreamQuote : IStreamQuote
         [JsonIgnore]
         public DateTime Time { get; set; }
 
+        /// <summary>
+        /// Called when the json message is deserialized.
+        /// We adjust the milliseconds by dividing it by 1,000,000 to reduce the time stamp from nanoseconds to milliseconds
+        /// </summary>
+        /// <param name="context">The context.</param>
         [OnDeserialized]
-        internal void OnDeserializedMethod(
-            StreamingContext context)
+        internal void OnDeserializedMethod(StreamingContext context)
         {
 #if NET45
-            Time = DateTimeHelper.FromUnixTimeMilliseconds(Timestamp);
+            Time = DateTimeHelper.FromUnixTimeMilliseconds((Int64)(Timestamp / 1000000));
 #else
-            Time = DateTime.SpecifyKind(
-                DateTimeOffset.FromUnixTimeMilliseconds(Timestamp)
-                    .DateTime, DateTimeKind.Utc);
+            Time = DateTime.SpecifyKind(DateTimeOffset.FromUnixTimeMilliseconds((Int64)(Timestamp / 1000000)).DateTime, DateTimeKind.Utc);
```

Comment:
I'm not sure why you need cast to Int64 here. If it compiles without this explicit cast - remove it, please.